draft-ietf-rats-corim.md

```diff
@@ -775,45 +775,54 @@ configuration files, read-only memory, fuses, IO ring configuration, partial
+Measurements can have class, instance or group scope.
+A `measurement-map`'s scope is determined by the `environment-map` it is paired with.
 
 Class measurements apply generally to all the Attesters in the given class.
 Instance measurements apply to a specific Attester instance.  Environments
 identified by a class identifier have measurements that are common to the
 class. Environments identified by an instance identifier have measurements that
 are specific to that instance.
 
-The supply chain entity that is responsible for providing the the measurements (i.e. Reference Values or Endorsed Values)
-is by default the CoRIM signer. If a different entity is authorized to provide measurement values,
-the `authorized-by` statement can be supplied in the `measurement-map`.
-
+The supply chain entity that is defined as the authority responsible for providing the the measurements (i.e. Reference Values or Endorsed Values)
+is by default the CoRIM signer.
+If different authorities may provide measurement values, the `"__authorities"` statement can be supplied in the `measurement-map`.
+If the `"__authorities"` statement is used within a condition, at least one of the listed authorities must match the respective ACS entry for the statement to match.
 
 ~~~ cddl
 {::include cddl/measurement-map.cddl}
 ~~~
 
+
 The following describes each member of the `measurement-map`:
 
-* `mkey` (index 0): An optional unique identifier of the measured
-  (sub-)environment.  See {{sec-comid-mkey}}.
+* `"__authorities"`: The cryptographic identities of individuals or organizations that are
+ designated authorities for this measurement. For example, producer of the measurement or a delegated supplier.
 
-* `mval` (index 1): The measurements associated with the (sub-)environment.
-  Described in {{sec-comid-mval}}.
+* `$measured-element-type-choice`: A map key distinct from `"__authorities"` that names a measured element. See {{sec-comid-mkey}}.
+
+* `measurement-values-map`: A map value that associates measurement values with the named measured element within the environment. Described in {{sec-comid-mval}}.
 
-* `authorized-by` (index 2): The cryptographic identity of the individual or organization that is
- the designated authority for this measurement. For example, producer of the measurement or a delegated supplier.
 
 ###### Measurement Keys {#sec-comid-mkey}
 
-The types defined for a measurement identifier are OID, UUID or uint.
+The types defined for identifying a measured element are uint, or a textual string different from `"__authorities"`.
 
 ~~~ cddl
 {::include cddl/measured-element-type-choice.cddl}
 ~~~
 
+A measured element serves is a local identifier within the scope of the Environment.
+
+A measured element MUST NOT be used as a refinement on the `environment-map`, since it names an element from a specific Environment's collection of measurements in one appraisal context.
```

Comment:
Why is this a bad thing?
When matching a reference value against the ACS, I was expecting the mkey to be used as part of finding measurement elements to be compared. Isn't that how the processing happens?

Reply:
I conceptualize environment map as a "place" and measurement-map as "properties", so I wouldn't say that a particular property from an attester should be considered more refinement on the place it's describing. If there is environment information, I think it should be in the transformation.